apps/babanews/pages/_app.tsx

```diff
@@ -168,26 +116,17 @@ function CustomApp({Component, pageProps}: CustomAppProps) {
               <meta name="msapplication-TileColor" content="#ffffff" />
               <meta name="theme-color" content="#ffffff" />
             </Head>
-
-            <Spacer>
+            <div className={hankenGrotesk.className}>
```

Comment:
Removal of the spacer now has the problem that if the page body isn't big enough, the footer isn't at the bottom of the page FYI

Reply:
this should be solved now as well, as I added spacer but in different place